includes/classes/order_total.php

```diff
@@ -63,6 +63,7 @@ function process() {
         $class = substr($value, 0, strrpos($value, '.'));
         if (!isset($GLOBALS[$class])) continue;
         $GLOBALS[$class]->process();
+        if (empty($GLOBALS[$class]->output)) continue; 
```

Comment:
Please, no single-line if statements.  Add braces, per current PSR.

Reply:
Roger that.